### PR TITLE
feat: update core schema v2 for alignments and mesh

### PR DESCRIPTION
**Pull request by Bento007**
## Description
Update core schema v2 to match the changes in for v1 from https://github.com/chanzuckerberg/cryoet-data-portal/issues/997

- remove is_portal_standard.

### CODE REVIEW

**Review by jgadling — COMMENTED** [review]
Please run `make-build-ingestion-config` to update the pydantic models before merging!

File: schema/core/v2.0.0/metadata.yaml
Comment (jgadling) [inline schema/core/v2.0.0/metadata.yaml:545]: Just calling this out because both old and new API's are missing field for tomogram cross references and dates. Is this PR intentionally adding functionality?

File: schema/core/v2.0.0/metadata.yaml
Comment (jgadling) [inline schema/core/v2.0.0/metadata.yaml:617]: We had a discussion about the `is_portal_standard` field the other day and iirc It's only supposed to be present on `Tomogram` and `Alignment`

Comment (jgadling) [reply schema/core/v2.0.0/metadata.yaml:617]: cc @uermel

**Review by jgadling — CHANGES_REQUESTED** [review]
Just marking this for changes while we get clarification on the crossreferences/dates fields on tomograms, and the `is_portal_standard` field on annotations. The rest lgtm!

File: schema/core/v2.0.0/metadata.yaml
Comment (Bento007) [reply schema/core/v2.0.0/metadata.yaml:545]: we do want these

File: schema/core/v2.0.0/metadata.yaml
Comment (Bento007) [reply schema/core/v2.0.0/metadata.yaml:617]: remove from v1 and v2

**Review by jgadling — APPROVED** [review]
Approved once the standardized field is removed from annotations

**Review by manasaV3 — APPROVED** [review]

**Pull request merged**
